app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package com.worldsnas.starplayer.view.musics_list
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.worldsnas.starplayer.core.BaseViewHolder
+import com.worldsnas.starplayer.databinding.ItemMusicBinding
+import com.worldsnas.starplayer.model.Music
+
+class MusicsListAdapter(private val context: Context, private val musics: ArrayList<Music>) :
+        RecyclerView.Adapter<BaseViewHolder>() {
+
+    inner class MusicListItemViewHolder(private val mBinding: ItemMusicBinding) : BaseViewHolder(mBinding.root), MusicItemViewModel.Listener {
+        private var mMusicViewModel: MusicItemViewModel? = null
+        override fun onBind(position: Int) {
```

Comment:
bejaye `position` mitonid khode object ro pass bedid `music` intori dige niazi be ham be inner bodan nist
toye viewHolder ham position ro darim niaz nist az biron begirim